scripts/lunchbot.js

```diff
@@ -0,0 +1,17 @@
+/* Commands:
+ *   lunchbot hi
+ *   lunchbot bye
+ */
+
+var leaveMsgs = [
+  'Bon appetit', 'Have a heavenly day', 'Goodbye gorgeous',
```

Comment:
should think of refining these by gender lol although i'd like to see lunchbot call paul gorgeous